afs/templates/views/components/reports/physical-thing.htm

```diff
@@ -184,21 +184,240 @@ <h1 class="afs-summary-report-title"><span data-bind="text: displayname" class="
 <div class="model-summary-report">
     <div class="afs-report-page">
         <!-- Names -->
-        <div data-bind="component: {
-            name: 'views/components/reports/scenes/name',
-            params: {
-                data: resource,
-                cards: nameCards
-            }
-        }"></div>
-        <!-- Description -->
-        <div class="margin-top-summary" data-bind="component: {
-            name: 'views/components/reports/scenes/description',
-            params: {
-                data: resource,
-                cards: descriptionCards,
-            }
-        }"></div>
+        <div class="afs-report-section">
+            <div class="afs-report-subsection">
+                <!-- ko ifnot: nameSummary() && nameSummary().length -->
+                <div class="afs-nodata-note">No names for this resource</div>
+                <!-- /ko -->
+
+                <!-- ko if: nameSummary() && nameSummary().length -->
+                <div class="afs-table pad-btm">
+                    <table class="table table-striped" cellspacing="0" width="100%">
+                        <thead>
+                            <tr class="afs-table-header">
+                                <th>{% trans "Name" %}</th>
+                                <th>{% trans "Name Type" %}</th>
+                            </tr>
+                        </thead>
+                        <tbody data-bind="dataTablesForEach: { data: nameSummary, dataTableOptions: getTableConfig(2) }">
+                            <tr>
+                                <td data-bind="text: content"></td>
+                                <td data-bind="text: type"></td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+                <!-- /ko -->
+            </div>
+        </div>
+        <!-- Image -->
+        <div class="afs-report-section">
+            <div class="afs-report-subsection">
+                <!-- ko ifnot: imageSummary() -->
+                <div class="afs-nodata-note">No images for this resource</div>
+                <!-- /ko -->
+
+                <!-- ko if: imageSummary() -->
+                <div class="afs-image-service-list-item" data-bind="foreach: imageSummary">
+                    <label data-bind = "text: displayname"></label>
+                    <span class="iiif-gallery-canvas-thumbnail">
+                        <img data-bind="attr: { src: thumbnail }">
+                    </span>
+                </div>
+                <!-- /ko -->
+            </div>
+        </div>
+        <!-- Statements -->
+        <div class="afs-report-section">
+            <div class="afs-report-subsection">
+                <!-- ko ifnot: statementsSummary() && statementsSummary().length -->
```

Comment:
Also, can you remove these "No statements/images/names" lines for the time being..  Namjun isn't using these in his summaries, so it appears inconsistent.